code/obj/flock/structure/flock_structure_parent.dm

```diff
@@ -43,7 +43,7 @@ ABSTRACT_TYPE(/obj/flock_structure)
 	last_process = TIME
 	health_max = health
 	time_started = world.timeofday
-	setMaterial(getMaterial("gnesis"))
+	setMaterial(getMaterial("gnesis"), appearance = FALSE)
```

Comment:
Other Flock stuff just uses mat_appearances_to_ignore = list("gnesis"), not sure if matters

Reply:
may as well be consistent 👍